### PR TITLE
Events Page Created Issue #60

### PR DESCRIPTION
**Pull request by maryam-olabisi**
All events page created.
Screenshot from 2020-01-20 22-58-01

Individual upcoming event details page 
Screenshot from 2020-01-20 22-59-54

Individual past event
Screenshot from 2020-01-20 23-01-43

### CODE REVIEW

**Comment by yemisi-o** [comment]
@maryam-olabisi Good job overall. 
1. The landing image for viewing a post looks stretched. Can you do something about the aspect ratio so the image doesn't look stretched?
2. The content still displays html tags (notice the <p> tag in the content). Please exclude them, you can check how this was handled on the blog post page
3. I think there should be more space after the description, before the event images title

File: models/Event.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	description: { type: Types.Markdown, initial: true, required: true, height: 100, markedOptions: { gfm: true } },
+	state: { type: Types.Select, options: 'draft, published, archived', default: 'draft', index: true },
+	author: { type: Types.Relationship, ref: 'User', index: true },
+	publishedDate: { type: Types.Date, index: true, dependsOn: { state: 'published' } },
+    image: { type: Types.CloudinaryImage, initial: true, required: true },
+    eventImages: { type: Types.CloudinaryImages },
+    eventDate: { type: Types.Date, initial: true, required: true },
+    location: { type: String, initial: true, required: true },
+    registrationLink: { type: String, initial: true, required: true },
+	pageView: { type: Types.Number, default: 0 },
```
Comment (yemisi-o) [inline models/Event.js]: @maryam-olabisi We don't need page views for the events page. Please remove

**Comment by yemisi-o** [comment]
@maryam-olabisi Also, please check mobile view for the event page. There seems to be an overflowing content.

In addition, please resolve merge conflicts

File: models/Event.js (hunk 1 above)
Comment (maryam-olabisi) [reply models/Event.js]: Removed

**Comment by maryam-olabisi** [comment]
Changes made, conflicts resolved.

**Pull request merged**